### PR TITLE
molecule 1.17.3 (new formula)

### PR DESCRIPTION
**Pull request by jvrplmlmn**
Signed-off-by: Javier Palomo Almena <javier.palomo.almena@gmail.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: formula/molecule.rb
Comment (woodruffw) [inline formula/molecule.rb]: Can we get a more detailed/in-depth test than this?

Comment (jvrplmlmn) [inline formula/molecule.rb]: If you could point me to some reference regarding testing for Formula I'd try to tackle it 😄 

For reference, I think that using the `molecule init` command will get us a better test scenario:

```
➜  molecule-testing molecule init
--> Initializing molecule in current directory...
Successfully initialized new role in /private/tmp/molecule-testing.
➜  molecule-testing ls -lrt
total 8
drwxr-xr-x 4 javier 136 Jan  8 17:51 tests
-rw-r--r-- 1 javier  55 Jan  8 17:51 playbook.yml
-rw-r--r-- 1 javier 439 Jan  8 17:51 molecule.yml
➜  molecule-testing tree
.
├── molecule.yml
├── playbook.yml
└── tests
    ├── test_default.py
    └── test_default.pyc

1 directory, 4 files
```

Don't know if its possible, but we can verify the scaffolding.

Comment (woodruffw) [inline formula/molecule.rb]: I don't think we currently have written docs on testing, but the `cmake` formula is a good example of a test.

> Don't know if its possible, but we can verify the scaffolding.

Yep, this is definitely possible. The `test` runs within a sandbox, so you can run `molecule init` and verify the existence of that tree.

**Comment by jvrplmlmn** [comment]
@woodruffw still waiting for CI (the last commit should be green), but I've added some missing dependencies and included tests for the Vagrant and Docker driver.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by woodruffw** [comment]
Thanks @jvrplmlmn. You can drop the old `--version` test :smile:

**Comment by jvrplmlmn** [comment]
@woodruffw done in 4c8c98b

**Comment by woodruffw** [comment]
Thanks! I'll let this sit for a few days for other maintainer feedback, but this LGTM :+1:

**Comment by dunn** [comment]
Mind squashing this to a single commit? Thanks!

**Comment by jvrplmlmn** [comment]
@dunn done

**Comment by dunn** [comment]
The bottle didn't get uploaded properly, so we've got to run CI once more.

**Comment by jvrplmlmn** [comment]
> The bottle didn't get uploaded properly, so we've got to run CI once more.

@dunn Any updates on this? Seems like https://bot.brew.sh/job/Homebrew%20Bottles/68334/console but since I'm not familiar with the Homebrew's CI, I don't actually know if everything is in order.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**
